contracts/Lender/FixLender.sol

```diff
@@ -35,20 +36,21 @@ contract FixLender is IFixLender, AccessControl {
     uint256 private immutable _poolEndDate;
     uint256 private immutable _minDeposit;
     uint256 private immutable _poolMaxLimit;
+    bytes32 public constant CLIENT_PORTAL = keccak256("CLIENT_PORTAL");
```

Comment:
Add the hash directly, this will slightly save some gas on deployment.
Same on flexible